src/examples/wasm/export/wasm.go

```diff
@@ -6,14 +6,21 @@ import (
 )
 
 func main() {
+	document := js.Global().Get("document")
+	updateCallback := js.FuncOf(func(this js.Value, args []js.Value) interface{} {
+		update()
+		return nil
+	})
+	document.Call("getElementById", "a").Set("oninput", updateCallback)
+	document.Call("getElementById", "b").Set("oninput", updateCallback)
+	update()
 }
 
 //go:export add
 func add(a, b int) int {
 	return a + b
 }
 
-//go:export update
```

Comment:
If this is no longer exported, this file is a poor example of the `export` method of interacting with webassembly :grimacing:

Reply:
Oh I see the `add` method is still exported, nevermind!